gh-pages/docs/community/projects.md

```diff
@@ -88,5 +88,10 @@ Please visit our <a href="https://catalog.datajoint.io/" target="_blank">Catalog
     <li><a href="https://isearch.asu.edu/profile/500553" target="_blank">Rick Gerkin, Arizona State University</a></li>
     <li><a href="https://www.pintolab.org" target="_blank">Lucas Pinto, Northwestern University</a></li>
     <li><a href="https://www.imbb.forth.gr/imbb-people/en/froudarakis-home" target="_blank">Emmanouil Froudarakis, FORTH</a> (<a href="https://github.com/ef-lab" target="_blank">GitHub</a>)</li>
+    <li>University of California, San Diego
+        <ul>
+        <li><a href="https://neurophysics.ucsd.edu" target="_blank">David Kleinfeld</a> (<a href="https://github.com/ActiveBrainAtlas" target="_blank">GitHub</a>)</li>
```

Comment:
Is this an embedded list with one item? My instinct is to only have an embedded list for multiple PIs/Labs; do you expect to find more from UCSD?

Reply:
Thanks @CBroz1.  Was actually thinking about creating embedded lists for all items in a subsequent pull request.  So that one could see all of the institutions at a quick glance.

Reply:
Sure. I'll keep that in mind for future updates